### PR TITLE
Added function for MR hypothesis tests

### PR DESCRIPTION
**Pull request by TomiSzucs**

### CODE REVIEW

File: cogstat/cogstat_hyp_test.py [hunk 1]
```diff
@@ -498,6 +498,73 @@ def variable_pair_hyp_test(data, x, y, meas_lev, normality=None, homoscedasticit
...
     return population_result
 
+def multiple_regression_hyp_tests(result, x, data, normality, homoscedasticity, multicollinearity):
+    """Hypothesis tests for model and regressor slopes in multiple linear regression.
+
+    Parameters
+    ----------
+    result : statsmodels regression result object
+        The result of the multiple regression analysis.
+    x : list of str
```
Comment (krajcsi) [inline cogstat/cogstat_hyp_test.py:508]: please, use another variable name

does the function work with one regressor? If so, please, extend the docstring. If not, please extend the code and the docstring :)

File: cogstat/cogstat_hyp_test.py [hunk 2]
```diff
@@ -498,6 +498,73 @@ def variable_pair_hyp_test(data, x, y, meas_lev, normality=None, homoscedasticit
...
+        List of explanatory variable names.
+    data : pandas dataframe
```
Comment (krajcsi) [inline cogstat/cogstat_hyp_test.py:510]: please, use this as the first parameter

File: cogstat/cogstat_hyp_test.py [hunk 3]
```diff
@@ -498,6 +498,73 @@ def variable_pair_hyp_test(data, x, y, meas_lev, normality=None, homoscedasticit
...
+        calculated or if the parameter was not specified.
+    homoscedasticity : bool or None
+        True when homoscedasticity is true, False otherwise. None if homoscedasticity could not be calculated or if
+        the parameter was not specified.
+    multicollinearity : bool or None
+        True when multicollinearity is suspected (VIF>10), False otherwise. None if the parameter was not specified.
+
+    Returns
+    -------
+    html text
```
Comment (krajcsi) [inline cogstat/cogstat_hyp_test.py:522]:
```suggestion
    str
        html text
```

**Review by krajcsi — CHANGES_REQUESTED** [review]
Looks good mostly!

File: cogstat/cogstat_hyp_test.py (hunk 2 above)
Comment (TomiSzucs) [reply cogstat/cogstat_hyp_test.py:510]: Parameter "data" is now first in line.

**Comment by krajcsi** [comment]
Looks good, thank you!

**Pull request merged**